test/lib/ansible_test/_data/requirements/sanity.changelog.in

```diff
@@ -1,3 +1,3 @@
-rstcheck < 4  # match version used in other sanity tests
+rstcheck < 6.0.0 # match version used in other sanity tests
```

Comment:
Hmm, apparently 4.0.0 to 5.x.y also has some quirks over 3.x.y:
```suggestion
rstcheck >= 4.0.0, < 6.0.0 # match version used in other sanity tests
```
and line 20 of test/sanity/code-smell/rstcheck.py needs to change to `        '-m', 'rstcheck.__init__',`.